core/pom.xml

```diff
@@ -22,6 +22,7 @@
         <netbeans.hint.license>apache20</netbeans.hint.license>        
         <maven.build.timestamp.format>yyyy-MM-dd'T'HH:mm:ssZ</maven.build.timestamp.format>        
         <builddate>${maven.build.timestamp}</builddate>        
+        <geotools.version>13.2</geotools.version>
```

Comment:
As this won't be in the core we can use java 8 like we do with the web module too.